modules/serverless-creating-broker-labeling.adoc

```diff
@@ -1,21 +1,23 @@
 // Module included in the following assemblies:
 //
-// * serverless/knative_eventing/serverless-using-brokers.adoc
+// * /serverless/develop/serverless-using-brokers.adoc
 
 :_content-type: PROCEDURE
 [id="serverless-creating-broker-labeling_{context}"]
 = Creating a broker by labeling a namespace
 
-You can create the `default` broker automatically by labeling a namespace that you own or have write permissions for.
+You can create the `default` broker automatically by labelling a namespace that you own or have write permissions for.
```

Comment:
You brought a bit o' the UK over to the US, eh? 😛

Reply:
lol I brought my laptop when I moved from Ireland so I guess the spellcheck is still UK English 😂 I'll switch it back, thanks!

Reply:
It's fine either way!